geometry/convex_hull.cpp

```diff
@@ -0,0 +1,69 @@
+#include "geometry/convex_hull.hpp"
+
+#include "geometry/robust_orientation.hpp"
+
+#include "base/stl_helpers.hpp"
+
+#include <algorithm>
+
+using namespace std;
+
+namespace m2
+{
+namespace
+{
+double const kEps = 1e-9;
+
+// Checks whether (p1 - p) x (p2 - p) > 0.
+bool IsCCW(PointD const & p1, PointD const & p2, PointD const & p)
+{
+  return robust::OrientedS(p1, p2, p) > kEps;
+}
+
+bool ContinuesHull(vector<PointD> const & hull, PointD const & p)
```

Comment:
The word "Continues" doesn't seem right here. Extends? FitsInto?
Also, consider swapping the arguments (or changing to passive voice: IsContinuedBy(hull, p)).

Reply:
IsContinuedBy SGTM